test/testsets/Allocations.jl

```diff
@@ -5,6 +5,6 @@
 @testset "Allocations" begin
     # ensure that there are 0 allocations for all the benchmarks
     for b in Benchmarks.all_benchmarks()
-        @test Benchmarks.runbenchmark(b, samples = 1, evals = 1).allocs == 0
+        @test Benchmarks.runbenchmark(b, samples = 1, evals = 1).allocs == 0 || "$b"
```

Comment:
Won't this allow allocations?

Reply:
I found this somewhere as a trick to display a custom error message on test failure (previously, an allocation test would fail without telling you which benchmark was causing allocations). I've still had runs where the allocation tests fail with this code, so I think it's ok.